src/qt/radiobut.cpp

```diff
@@ -59,3 +77,44 @@ QWidget *wxRadioButton::GetHandle() const
 {
     return m_qtRadioButton;
 }
+
+void wxRadioButton::createAndJoinNewGroup( wxWindow *parent )
+{
+    m_qtButtonGroup = new QButtonGroup( parent->GetHandle() );
+    m_qtButtonGroup->addButton( m_qtRadioButton );
+}
+
+void wxRadioButton::searchForPreviousGroupToJoin( wxWindow *parent )
+{
+    wxWindowList::compatibility_iterator node = parent->GetChildren().GetLast();
+
+    for ( ; node; node = node->GetPrevious() )
+    {
+        wxWindow *child = node->GetData();
```

Comment:
very minor: an easier to read name for this variable is IMHO `previous` or something along that line.